sci-mathematics/octave/octave-5.1.0.recipe

```diff
@@ -3,16 +3,21 @@ DESCRIPTION="GNU Octave is a high-level language, primarily intended for \
 numerical computations.  It provides a convenient command line interface \
 for solving linear and nonlinear problems numerically."
 HOMEPAGE="http://www.octave.org/"
-COPYRIGHT="1996-2017 John W. Eaton"
+COPYRIGHT="1996-2019 John W. Eaton"
 LICENSE="GNU GPL v3"
-REVISION="5"
+REVISION="1"
 SOURCE_URI="https://ftp.gnu.org/gnu/octave/octave-$portVersion.tar.gz"
-CHECKSUM_SHA256="80c28f6398576b50faca0e602defb9598d6f7308b0903724442c2a35a605333b"
+CHECKSUM_SHA256="e36b1124cac27c7caa51cc57de408c31676d5f0096349b4d50b57bfe1bcd7495"
 PATCHES="octave-$portVersion.patchset"
 
 ARCHITECTURES="!x86_gcc2 x86 x86_64"
 SECONDARY_ARCHITECTURES="?x86"
 
+liboctaveVersion="7.0.0"
+liboctaveVersionCompat="$liboctaveVersion compat >= ${liboctaveVersion%%.*}"
+liboctinterpVersion="7.0.0"
+liboctinterpVersionCompat="$liboctinterpVersion compat >= ${liboctinterpVersion%%.*}"
```

Comment:
If both versions are the same you can just use libVersion and libVersionCompat?

Reply:
True, but the current solution is also ok.